dev/docker/compose-v3

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -eou pipefail
+
+docker compose \
+  -f dev/docker/docker-compose.yml \
+  --env-file dev/docker/local.env \
```

Comment:
I dont think that you need the env file. Its for d14n

Reply:
Ah saw this after I merged, I'll just remove the line in another line of my PRs